remix-debug/src/debugger/stepManager.js

```diff
@@ -122,7 +122,10 @@ class DebuggerStepManager {
 
   stepOverForward (solidityMode) {
     if (!this.traceManager.isLoaded()) return
-    var step = this.traceManager.findStepOverForward(this.currentStepIndex)
+    let scope = this.debugger.callTree.findScope(this.currentStepIndex)
```

Comment:
I think `step` should default to `this.currentStepIndex` if `findScope` does not return anything

Reply:
set `var step = this.currentStepIndex + 1`